docs/application/native/guides/location-sensors/user-awareness.md

```diff
@@ -0,0 +1,155 @@
+    ```
+    The ```__presence_detected_cb()``` function is a callback, which is called when the user presence is detected.
+  
+2.  Using the user presence detection consumes the system resources, so if the service is not used, stop the user presence detection using the ```ua_monitor_stop_presence_detection()``` API. Call the ```ua_monitor_start_presence_detection()``` API again if user presence detection is needed.
+    ```
+    ua_monitor_stop_presence_detection(g_ua_mon_h);
+    ```
+  
+## Start the absence detection
+
+To start the absence detection, follow these steps:
+  
+1.  Start the user absence detection using the `ua_monitor_start_absence_detection()` API. This call is asynchronous and only initiates the process of starting the user absence detection. Once the absence detection is started, the registered callbacks are invoked when their corresponding events take place. To know when the user absence is detected, use the `ua_absence_detected_cb()` function:
+    ```
+    ua_monitor_start_absence_detection(g_ua_mon_h, NULL, detection_mode, __absence_detected_cb, NULL);
+    ```
+    The `__absence_detected_cb()` is a callback function, which is called when the user absence is detected. 
+    
+2.  Using the user absence detection consumes the system resources, so if the service is not used, stop the user absence detection using the `ua_monitor_stop_absence_detection()` API. Call the `ua_monitor_start_absence_detection()` API again if user presence detection is needed:
+    ```
+    ua_monitor_stop_absence_detection(g_ua_mon_h);
+    ```
+ 
+  
+## Start the location detection
+
+To start the location detection, follow these steps:
+
+1.  Start the user location detection using the `ua_monitor_start_location_detection()` API. This call is asynchronous and only initiates the process of starting the user location detection. Once the location detection is started, the registered callbacks are invoked when their corresponding events take place. To know when the user location is detected, use the `ua_location_detected_cb()` function:
+    ```
+    ua_monitor_start_location_detection(g_ua_mon_h, NULL, detection_mode, __location_detected_cb, NULL);
+    ```
+    The `__location_detected_cb()` is a callback function, which is called when the user location is detected.
+    
+2.  Using the user location detection consumes the system resources, so if the service is not used, stop the `ua_monitor_stop_location_detection()` API. Call the `ua_monitor_start_location_detection()` API again if user location detection is needed: 
+    ```
+    ua_monitor_stop_location_detection(g_ua_mon_h);
+    ```
+  
+## Release all resources
+At the end of application, destroy all used resources (such as, user handle, service handle, device handle, etc.) as below:
+    
+    ```
+    ua_device_destroy(g_device_h);
+    ua_user_destroy(g_user_h);
+    ua_service_destroy(g_service_h);
+    ua_monitor_destroy(g_ua_mon_h);
+    ```
```

Comment:
Hi @mijin85cho , hope you're well. There seems to be an issue in the staging site for these lines. 

The code appears to be alright, but the code tags are visible in the code block on the staging site. 

I have provided the necessary suggestions, and the author has committed them but there seems to be no change. Can you please suggest what can be done?

![image](https://user-images.githubusercontent.com/113158244/200802935-040fec8a-1894-4c8f-8891-aaba424534a9.png)

Reply:
Hello, @maliha-srbd I hope you the same.

I think it is the same issue with https://github.com/Samsung/tizen-docs/pull/1711#discussion_r1006402483.
Would you please check the comment? I think the spaces before the code block should be removed.

Reply:
Hello @mijin85cho , thank you for your response. 

I tried removing the spaces before the code blocks, but for some reason the suggestion is not working. Could you please suggest how I could approach this issue?

![ss1](https://user-images.githubusercontent.com/113158244/201000354-762eaa5c-a606-43c5-9b77-f6c6371d60a5.png)

Here's a screenshot of the preview. The suggested change is not being implemented by GitHub. Could you please suggest how I could approach this issue?

![ss2](https://user-images.githubusercontent.com/113158244/201000366-71f327fb-726b-49da-8239-f145fa293f7e.png)